detox/test/e2e/25.statusbar.test.js

```diff
@@ -0,0 +1,170 @@
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+describe('StatusBar Override', () => {
+  it('Time', async () => {
```

Comment:
Why there is no platform-specific marker for `:ios:` in these e2e tests?

Reply:
Fixed